docker/Dockerfile

```
@@ -140,6 +142,9 @@ RUN chmod u+x /entrypoint.sh
 
 # Entrypoint environment variables
 ENV ENTRYPOINT_FEATURES=${ENTRYPOINT_FEATURES}
+# We repeat the ARGs here, so they are available in the entrypoint.sh script for $RUN_ALL_EXPERIMENTAL_TESTS
+ARG EXPERIMENTAL_FEATURES="elasticsearch journald prometheus filter-reload"
```

```suggestion
ARG EXPERIMENTAL_FEATURES="shielded-scan"
```

Some tests rely on these features being disabled.

---

These are the test failures:
- `config_tests`: expects config fields that aren't set, we should fix this to emit a warning instead of panicking
- `end_of_support_is_checked_at_start`: The logs with these features are unexpected or the timeout becomes too short, it should be enough for this test to pass without extra features
- `ephemeral_missing/existing_directory`/`misconfigured_ephemeral_missing/existing_directory`: the command exits without a kill where the test expected a kill exit, seemingly because it expects the http request for inserting data into elasticsearch to succeed and panics early since there's no elasticsearch server.
- `persistent_mode`/`sync_large_checkpoints_x`/`start_args`/`tracing_endpoint`: Have the same issue as the ephemeral directory tests, the spawned process exits before doing what's expected
    
These tests expect the second zebrad testchild to fail, but either could open port 9200 or fail first: 
- zebra_metrics_conflict
- zebra_rpc_conflict
- zebra_state_conflict
- zebra_tracing_conflict